packages/typescript-estree/tests/lib/convert.ts

```diff
@@ -188,4 +188,41 @@ describe('convert', () => {
     );
     checkMaps(ast);
   });
+
+  it('should correctly create node with range and loc set', () => {
```

Comment:
within a code we there is no location set, we always rely on automatic generation for it